dev/firely/README.md

```diff
@@ -0,0 +1,38 @@
+# Firely Server
+Configuration required to deploy a review version of Firely server
+
+## Setup
+Copy the default env files:
+
+    for file in *.default; do
+        cp "$file" "${file%%.default}"
+    done
+
+Copy the `.env` file default:
+
+    cp default.env .env
+
+Modify any `.env` files as necessary. Lines that are not commented-out are required, commented lines are optional.
+
+## Deploy
+To pull the latest configured docker images, and re-deploy services as necessary, run the following command:
+
+    docker-compose pull && docker-compose up --detach
+
+Note that the firely app may fail to connect to the database when both are started at the same time.
+
+### Licensing
```

Comment:
I had issues when running `docker-compose up -d` before installing the license: docker-compose will create a root-owned empty file if the license file isn't there when `docker-compose up -d`.

I'd suggest moving the license file install instructions before the `docker-compose` steps to avoid that

Passing [the license as an environment variable](https://docs.fire.ly/projects/Firely-Server/en/4.9.0/deployment/docker.html#providing-license-via-an-environment-variable) could also be another option